linkerd/docs/routers.md

```diff
@@ -0,0 +1,143 @@
+<a name="routers"></a>
+# Routers
+
+All configurations must define a **routers** key, the value of which
+must be an array of router configurations. Routers also include **servers**, which define their entry points, and **client**, which configures how clients are built.
+
+## Router Parameters
+
+<aside class="notice">
+These parameters are available to the router regardless of protocol. Routers may also have protocol-specific parameters.
+</aside>
+
+```yaml
+routers:
+- protocol: http
+  servers: ...
+  client: ...
+  announcers: ...
+  bindingCache: ...
+  label: myPackIce
+  dstPrefix: /walruses/http
+  baseDtab: |
+    /host                => /#/io.l5d.fs;
+    /walruses/http/1.1/* => /host;
+  failFast: false
+  timeoutMs: 10000
+  bindingTimeoutMs: 5000
+  responseClassifier: io.l5d.nonRetryable5XX
+```
+
+Key | Default Value | Value Description
+--- | ------------- | -----------------
+protocol | _required_ | either "[http](#http-1-1-protocol)", "[thrift](#thrift-protocol)", or "[mux](#mux-protocol-experimental)"
+servers | _required_ | a list of [server objects](#servers)
```

Comment:
I was noticing that too. Sometimes the descriptions are full sentences and sometimes they are only subjects/noun phrases. We could probably pretend they're all sentences though